### PR TITLE
Meta Chunker: update the option on post deletion

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/79593

This PR makes sure proper cleanup is in place when a post is deleted

### CODE REVIEW

File: src/Tribe/Meta/Chunker.php [hunk 1]
```diff
@@ -846,4 +847,25 @@ public function get_key_option_name() {
...
+	/**
+	 * Removes the entries associated with a deleted post from the cache and the database option.
+	 *
+	 * @param int $post_id A post ID
+	 */
+	public function remove_post_entry( $post_id ) {
+		$this->prime_chunks_cache();
+
+		foreach ( $this->chunks_cache as $key => $value ) {
+			if ( 0 === strpos( $key, '' . $post_id ) ) {
```
Comment (bordoni) [inline src/Tribe/Meta/Chunker.php]: Why append an empty thing instead of casting it as string?

Comment (lucatume) [inline src/Tribe/Meta/Chunker.php]: faster usually but really makes no difference here

Comment (bordoni) [inline src/Tribe/Meta/Chunker.php]: Up to you ;)

Comment (lucatume) [inline src/Tribe/Meta/Chunker.php]: Let's go with clarity (your option)

**Pull request merged**